integration/eth2network/pos_eth2_network.go

```diff
@@ -177,8 +177,8 @@ func (n *PosImpl) Start() error {
 
 func (n *PosImpl) Stop() error {
```

Comment:
let's also swap `CheckHostRPCServersStopped` and `StopEth2Network` in `TearDown`.

To make sure that the l2 stuff is truly dead before killing the l1 stuff